juntagrico/static/js/juntagrico.js

```diff
@@ -65,21 +65,47 @@ function move_email_button() {
     $("form#email-sender").appendTo("#filter_header div:first-child");
 }
 
-function email_submit() {
-    $("form#email-sender").submit(function (event) {
-        var emails = [];
-        $("#filter-table").find("tr").each(function () {
+ function fetch_emails() {
+    var emails = []
+     $("#filter-table").find("tr").each(function () {
             var txt = $(".email", this).text().trim();
             if (txt.length > 0)
-                emails.push(txt);
+                for(var email of txt.split(",")){
+                    emails.push(email.trim());
+                }
         });
+    return emails;
+ }
+ function email_submit() {
+    $("form#email-sender").submit(function (event) {
+        var emails = fetch_emails();
+
         $("#recipients").val(emails.join("\n"));
         $("#recipients_count").val(emails.length);
         return;
     });
 }
 
+function get_sb_config() {
+    var sb_columns = true;
+    if(typeof search_builder_enabled !== 'undefined' && search_builder_enabled){
+        sb_columns = search_builder_columns;
+    }
+    return {
+        columns: sb_columns
+    };
+}
+
+function get_dom(){
+    var dom_text = '';
```

Comment:
The previously used global search field won't show anywhere with this dom configuration. Is that intended?
Also it seems like the Bootstrap integration changes the default dom and by overwriting that default the bootstrap integration is lost: https://datatables.net/reference/option/dom#Styling

Reply:
i struggled to understand the styling part but since in the download section i configured datatables for bootstrap it makes sense it also adds the bootsrap classes for formcontrols
yes the searchfield is gone i think that is ok since i do not think it is usefull in this context

Reply:
yes by configuring the download for bootstrap the default value of dom_text is adjusted to wrap the table into another row and col-sm-12. By setting the dom_text to an empty string this is overwritten. However I think this is not an issue, as we already have the table in a row-col wrapping. Let's open an issue if it becomes one.